/**:
  ros__parameters:
    drone:
      m: 3.0
      l: 0.265
      MoiArray: [0.060, 0.060, 0.080]
      motor_const: 1.465e-7
      moment_const: 0.01569
    l1_adaptive:
      dob_looptime: 0.010
      As_array: [-5.0, -5.0, -5.0, -5.0, -5.0, -5.0]
      freq_cutoff_trans: 20.0
      freq_cutoff_rot: 60.0
    lpf:
      lin_vel_cutoff: 20.0
      ang_vel_cutoff: 60.0